cfme/intelligence/reports/reports.py

```diff
@@ -312,6 +313,42 @@ def delete(self, cancel=False):
     def saved_reports(self):
         return self.collections.saved_reports
 
+    def create_schedule(self, name=None, description=None, active=None,
```

Comment:
Required:
Why all argument default value `None`?

Reply:
That is in the case when no argument value is passed in the data.